libsbom/spdx.py

```diff
@@ -0,0 +1,211 @@
+import logging
+import datetime
+import uuid
+
+from spdx_tools.spdx.writer.json     import json_writer
+from spdx_tools.spdx.writer.tagvalue import tagvalue_writer
+from spdx_tools.spdx.writer.xml      import xml_writer
+from spdx_tools.spdx.writer.yaml     import yaml_writer
+from spdx_tools.spdx.model import (
+    Actor,
+    ActorType,
+    Annotation,
+    AnnotationType,
+    Checksum,
+    ChecksumAlgorithm,
+    CreationInfo,
+    Document,
+    ExternalPackageRef,
+    ExternalPackageRefCategory,
+    Package,
+    PackagePurpose,
+    Relationship,
+    RelationshipType
+)
+
+from cas_wrapper import CasWrapper
+from version import __version__
+
+ALMAOS_VENDOR = 'AlmaLinux OS Foundation'
```

Comment:
I'd say that anything duplicated like `ALMAOS_VENDOR`, or anything that is subject to be shared between different components of the project can be moved into a `constants.py` file

Reply:
I just added a commit that moves shared constants (and stuff that isn't shared yet but might be in the future) to `libsbom/constants.py`.